src/lib.rs

```diff
@@ -102,3 +109,7 @@ pub use crate::errors::ErrorKind;
 pub use crate::errors::Result;
 #[doc(inline)]
 pub use crate::kv::{BoundRange, Key, KvPair, ToOwnedRange, Value};
+#[doc(inline)]
+pub use crate::raw::{Client as RawClient, ColumnFamily};
```

Comment:
Is this move purely for discovery purposes? I think it's logical. :)

Reply:
It's an effort to have a simpler module structure with fewer re-exports and fewer deep paths. I don't think it is optimal yet, but you should be able to use the client for the most common tasks with only the top-level items.